apps/web/src/app/setup/page.tsx

```diff
@@ -0,0 +1,65 @@
+'use client'
```

Comment:
Is better to move this to an inner component and let the routes always be server side rendered